api/models.py

```diff
@@ -1462,13 +1462,13 @@ def __str__(self):
 
 class ActionOrg:
     NATIONAL_SOCIETY = 'NTLS'
-    FOREIGN_SOCIETY = 'PNS'
+    RCRC = 'RCRC'
```

Comment:
I think we should not change this, we should only change the label

Reply:
Are RCRC and FOREIGN_SOCIETY the same?